src/node_builtins.cc

```diff
@@ -710,6 +734,11 @@ void BuiltinLoader::RegisterExternalReferences(
   registry->Register(GetCacheUsage);
   registry->Register(CompileFunction);
   registry->Register(HasCachedBuiltins);
+  auto internalized_builtins = GetInternalizedBuiltinSourceMap();
+  for (auto const& x : *internalized_builtins) {
```

Comment:
Actually I wonder if we really have to have two maps and two classes? The externalized builtins (that need non-static external string sources) are all known (see the builtin loader constructor), so we could just explicitly skip them here, that makes the difference of destruction go away, and we don't have to check two maps in all the code above, so the code could be much simpler.

Reply:
Resolving this conversation since there is now only one map/class. (Although I took a slightly different approach by just generating the registration statements in JS2C, rather than skipping externalized builtins.)